fs/dafilesrv/dafilesrv.cpp

```diff
@@ -524,6 +528,12 @@ int main(int argc, const char* argv[])
         IPropertyTree *daFileSrv = env->queryPropTree(dafilesrvPath);
         Owned<IPropertyTree> _dafileSrv;
 
+        // merge in bare-metal global expert settings
+        IPropertyTree *globalExpert = nullptr;
+        globalExpert = env->queryPropTree("Software/Globals");
+        if (globalExpert)
+            mergePTree(expert, globalExpert);
```

Comment:
I think this should be synchronizePTree(expert, globalExpert, false, true);

(and the other mergePTree)

In reality it won't make any practical difference if expert contains attributues only, but mergePTree adds properties to the target, so if there are property sub-trees in expert, then it would add them each time, causing duplicates.

Reply:
ok, good point.
I've changed all 3 mergePTree() to synchronizePTree().
But the first one needs rootsMustMatch arg to be false because "expert" != "Software/Globals"
I've just amended the sole commit because its a small change.